docs/user-guide/compute-resources.md

```diff
@@ -269,6 +269,86 @@ LastState: map[terminated:map[exitCode:137 reason:OOM Killed startedAt:2015-07-0
 
 We can see that this container was terminated because `reason:OOM Killed`, where *OOM* stands for Out Of Memory.
 
+## Alpha Features
```

Comment:
Are you using the GHFM preview or viewing the netify rendered output (see https://deploy-preview-1783--kubernetes-io-vnext-staging.netlify.com/docs/user-guide/compute-resources/)